tools/run_cargo_generate-lockfile.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Runs `cargo generate-lockfile` (which generates and/or updates the Cargo.lock
+# file for a crate) on every subdirectory from . that has a Cargo.toml file.
+#
+# Author: Brad Campbell <bradjc5@gmail.com>
+
+FAIL=0
+
+set -e
+
+# Verify that we're running in the base directory
+if [ ! -x tools/run_cargo_generate-lockfile.sh ]; then
+	echo ERROR: $0 must be run from the tock repository root.
+	echo ""
+	exit 1
+fi
+
+for f in $(find . | grep Cargo.toml); do
```

Comment:
Wouldn't `find . -name Cargo.toml` work more directly?

Reply:
Goes way back to https://github.com/tock/tock/commit/bf8ca045bcd00dad34b5c93f57b0da3d879e590e and I'm always nervous to change these things for cross platform reasons. But if it is tested and works we can change it.

Reply:
Makes sense to keep it with `grep` then.